migrationmanager/migrators/logs/migrations/000014_new_schema.up.sql

```diff
@@ -0,0 +1,119 @@
+CREATE TABLE IF NOT EXISTS  signoz_logs.logs_v2_resource_bucket ON CLUSTER {{.SIGNOZ_CLUSTER}}
+(
+    `labels` String CODEC(ZSTD(5)),
+    `fingerprint` String CODEC(ZSTD(1)),
+    `seen_at_ts_bucket_start` Int64 CODEC(Delta(8), ZSTD(1)),
+    INDEX idx_labels lower(labels) TYPE ngrambf_v1(4, 1024, 3, 0) GRANULARITY 1
+)
+ENGINE = ReplacingMergeTree
+PARTITION BY toDate(seen_at_ts_bucket_start / 1000)
+ORDER BY (labels, fingerprint, seen_at_ts_bucket_start)
+SETTINGS ttl_only_drop_parts = 1, index_granularity = 8192;
+
+
+CREATE TABLE IF NOT EXISTS  signoz_logs.distributed_logs_v2_resource_bucket ON CLUSTER {{.SIGNOZ_CLUSTER}}
+(
+    `labels` String CODEC(ZSTD(5)),
+    `fingerprint` String CODEC(ZSTD(1)),
+    `seen_at_ts_bucket_start` Int64 CODEC(Delta(8), ZSTD(1))
+)
+ENGINE = Distributed('cluster', 'signoz_logs', 'logs_v2_resource_bucket', cityHash64(labels, fingerprint, seen_at_ts_bucket_start));
```

Comment:
Why do we have `seen_at_ts_bucket_start` in sharding keys? If we just use a fingerprint it will guarantee the same fingerprint goes to the same shard.

Reply:
because there will be a comparison filter for `seen_at_ts_bucket_start` in all the queries.

Reply:
Not clear, how does comparison filter in query necessitate that it needs to be part of the sharding key?

Reply:
Oh, my bad I took the sharding key for `order by`. 

Yeah right `seen_at_ts_bucket_start` doesn't seem to add any value to the sharding key. @raj-k-singh  any specific reason for adding this, else we can remove.